src/components/PlantingSteps/PlantingSteps.tsx

```diff
@@ -3,20 +3,39 @@ import React from 'react';
 import { useStyles } from 'react-native-unistyles';
 import { useLocalSearchParams } from 'expo-router';
 
+import { PlantingStep as TPlantingStep } from '@src/api/model';
 import { useAPIGetPlantingSteps } from '@src/api/plants/plants';
+import { queryKeys } from '@src/constants/queryKeys';
 import { PlantingStep } from '../PlantingStep/PlantingStep';
 import { styleSheet } from './PlantingSteps.style';
+import { TStapDay } from '../PlantCareBody/PlantCareBody';
 
-export const PlantingSteps = () => {
+type Props = {
+	curStepDay: TPlantingStep | null;
+	handlerCurStepDay: (day: TStapDay) => void;
+};
+
+export const PlantingSteps = ({ curStepDay, handlerCurStepDay }: Props) => {
 	const { styles } = useStyles(styleSheet);
 	const { uuid } = useLocalSearchParams<{ uuid?: string }>();
 
-	const { data } = useAPIGetPlantingSteps(uuid ?? '');
+	const plantUuid = uuid ?? '';
+
+	const { data } = useAPIGetPlantingSteps(plantUuid, {
+		query: {
+			queryKey: [queryKeys.GET_PLANT_STEPS, plantUuid],
+		},
+	});
```

Comment:
I don't think we need to execute useAPIGetPlantingSteps when we have no uuid

Reply:
Did you use Storybook to create this component?

Reply:
no i don't use Storybook